### PR TITLE
[COM-51] Implement skeleton Point Mass ("Particle") Physics Simulation

### PR DESCRIPTION
**Pull request by imnorookie**
Added a skeleton to simulate _point masses_ — hereafter erroneously referred to as _particles_ as is standard practice in Game Dev (distinction made not to be pedantic but for clarity should one need to research the physics of it: google point masses).

Efforts were made to comment functions to add more detail from the textbook. Theory and other documentation that isn't fully covered can be found [here](https://publish.obsidian.md/shadow-moses/Physics/Overview).

### CODE REVIEW

**Review by jharris99-git — DISMISSED** [review]

File: Physics/particle.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#include "core.h"
+
+namespace AE86 {
+	/**
+	 * A particle is the simplest object that can be simulated in the
+	 * physics system. In reality, it is a point mass as per Newton's
+	 * Laws of motion.
+	 * Author: Jas Singh
+	*/
+	class Particle {
```
Comment (le-rich) [inline Physics/particle.h:10]: I can see this confusing people down the road as a class name, but it should be fine.

File: Physics/core.h
Comment (le-rich) [inline Physics/core.h:1]: May want to put headers in a folder called include

File: Physics/particle.cpp
Comment (le-rich) [inline Physics/particle.cpp:1]: May want to put source files in a folder called src

**Review by le-rich — CHANGES_REQUESTED** [review]
Comments describe changes

File: Physics/particle.cpp
Comment (imnorookie) [reply Physics/particle.cpp:1]: Okie! Donezo :3333

File: Physics/core.h
Comment (imnorookie) [reply Physics/core.h:1]: VeWey astuWute of yuWu!! Done!! ~~~

File: Physics/particle.h (hunk 1 above)
Comment (imnorookie) [reply Physics/particle.h]: Renamed to PointMass XDD

**Review by le-rich — APPROVED** [review]
LGTM daddy

**Review by jchouhan19 — COMMENTED** [review]
Checked over. Also appreciate the name change for clarity

**Review by jchouhan19 — DISMISSED** [review]
Checked and Approved

**Review by jharris99-git — APPROVED** [review]
Self-review, small change need 1 more approval to finish this.

**Comment by imnorookie** [comment]
> Self-review, small change need 1 more approval to finish this.

Thanks for catching that and adding damping correctly.

**Pull request merged**